BSP/STM32F413/Src/BSP_GPIO.c

```diff
@@ -88,7 +88,7 @@ uint8_t BSP_GPIO_Read_Pin(port_t port, uint16_t pinmask){
  * @return  None

```

Comment:
documentation needs to reflect that the boolean true = ON and boolean false = OFF